fcu_sim/launch/spawn_mav.launch

```diff
@@ -19,7 +19,7 @@
 
   <!-- send the robot XML to param server -->
   <param name="robot_description" command="
-    $(find xacro)/xacro.py '$(arg model)'
+    $(find xacro)/xacro --inorder '$(arg model)'
```

Comment:
We actually need to hang on to this (I know it gives a warning) because of compatibility with indigo.